.nuspec/Microsoft.Maui.Controls.props

```diff
@@ -5,6 +5,20 @@
 		<DefineConstants Condition="!$(_IsHotRestartDefined) And '$(IsHotRestartBuild)' == 'true'">HOTRESTART;$(DefineConstants)</DefineConstants>
 	</PropertyGroup>
 
+	<PropertyGroup Condition=" '$(SingleProject)' == 'true' and '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'windows' ">
+		<OutputType Condition="'$(OutputType)' == 'Exe'">WinExe</OutputType>
+		<EnablePreviewMsixTooling Condition="'$(EnablePreviewMsixTooling)' == ''">true</EnablePreviewMsixTooling>
+		<_SingleProjectRIDRequired Condition="'$(OutputType)' == 'WinExe'">true</_SingleProjectRIDRequired>
+		<_SingleProjectRIDSpecified Condition="'$(RuntimeIdentifier)' != '' or '$(RuntimeIdentifiers)' != ''">true</_SingleProjectRIDSpecified>
+	</PropertyGroup>
+
+	<PropertyGroup Condition=" '$(SingleProject)' == 'true' and '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'windows' and '$(_SingleProjectRIDRequired)' == 'true' and '$(_SingleProjectRIDSpecified)' != 'true' ">
+		<_SingleProjectHostArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)</_SingleProjectHostArchitecture>
+		<_SingleProjectHostArchitecture>$(_SingleProjectHostArchitecture.ToLower())</_SingleProjectHostArchitecture>
+		<RuntimeIdentifier>win10-$(_SingleProjectHostArchitecture)</RuntimeIdentifier>
+		<_MauiUsingDefaultRuntimeIdentifier>true</_MauiUsingDefaultRuntimeIdentifier>
+	</PropertyGroup>
```

Comment:
Should/could this be in some Sdk.props that is imported? Should this whole file be in a Sdk.props?

Reply:
@jonathanpeppers or @rolfbjarne I saw a Sdk.props in the iOS workloads and it looks like we import this in the sample place, just less "officially".

Reply:
Workloads don't actually have an `Sdk.props`... There is an `AutoImport.props`, but you are not supposed to set properties in there -- only item groups.

If you want to apply some WinUI-specific settings we can put them here:

https://github.com/dotnet/maui/blob/main/src/Workload/Microsoft.Maui.Sdk/Sdk/WinUI.targets

But this runs after the `.csproj` file.

Reply:
IOS has a sdk.props, but it is a weird lie that runs as a target. I can't use the WinUI.targets as it runs after the nugets. However, the sdk.props runs after the csproj but before the nugets. Basically at the same point as this file.